agent-packaging/packaging-scripts/build_packages_deb.sh

```diff
@@ -15,22 +15,15 @@
 
 set -e
 
-function exit_error
-{
-  echo "build failed"
-  exit 1
-}
+echo "started build..."
 
-trap exit_error ERR
+apt-get -y update && apt-get -y upgrade
+apt-get install -y git-core
+git clone --branch agent-build "https://github.com/GoogleCloudPlatform/osconfig.git"
 
-URL="http://metadata/computeMetadata/v1/instance/attributes"
-GCS_PATH=$(curl -f -H Metadata-Flavor:Google ${URL}/daisy-outs-path)
-BASE_REPO=$(curl -f -H Metadata-Flavor:Google ${URL}/base-repo)
+cd osconfig
 
-apt-get install -y git-core 
-git clone "https://github.com/${BASE_REPO}/compute-image-tools.git"
-cd compute-image-tools/cli_tools/google-osconfig-agent 
-packaging/setup_deb.sh 
-gsutil cp /tmp/debpackage/google-osconfig-agent*.deb "${GCS_PATH}/" 
+source ./agent-packaging/packaging-scripts/setup_deb.sh
+gsutil cp /tmp/debpackage/google-osconfig-agent*.deb "gs://osconfig-agent-package/"
```

Comment:
two nits, the hardcoded gs path and a top level bucket just for osconfig. i think we will probably want some hierarchy for builds so we can get every built artifact while also having a 'destination' bucket with folders for things which should be injected to repos. as written we'd always overwrite for new builds of same version.